src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -644,6 +644,15 @@
         "Power Topology": ["FeatureMap"],
         "Valve Configuration and Control": ["RemainingDuration"],
         "Boolean State Configuration": ["CurrentSensitivityLevel"],
+        "Water Heater Management": [
+            "HeaterTypes",
+            "HeatDemand",
+            "TankVolume",
+            "EstimatedHeatRequired",
+            "TankPercentage",
+            "BoostState",
+            "FeatureMap"
```

Comment:
Add ClusterRevision here too, since you are handling it via AttributeAccessInterface now?

Reply:
Change made. Files regenerated.